extensions/community_extensions/Protected Audience Support.md

```diff
@@ -0,0 +1,125 @@
+# Bid Request Values
+
+## Object: InterestGroupAuctionSupport 
+
+This object allows sellers to signal interest group auction support for an Imp. 
+
+<table>
+  <tr>
+    <td><strong>Attribute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>ae</code></td>
+    <td>integer;<br>default 1</td>
+    <td>Interest Group auction environment support for this impression: <br>
+1 = on-device orchestrated auction<br>
+2 = server-orchestrated auction<br>
```

Comment:
This should be 3 to match in place stuff

Reply:
sorry...what do you want me to add for this one?

Reply:
change the 2 to 3, add a zero

Reply:
what should the value for 0 be? just no IG?